### ISSUE TITLE
GUI way to generate a new query

### ISSUE THREAD

**Issue by bittib010** [amestofortytwo/InteractiveKQLCheatsheet#26]
Makes it easier to contribute without coding knowledge.

Idea is to retrieve indexes of the query and make the user input the popup text and a input csv format for tags.

**Comment by remiks** [comment]
I think this needs to create one separate json file for each query that is added. OR, if you can make a method that controls one big file..(?)

**Comment by bittib010** [comment]
How about making it only generate the code which then will be added to a PR manually? To keep records and filtering in order to avoid false information..?

I was thnking of starting out the logic towards generating the json and them based on how far weve come see what solution fits best; generate new json, control bigger file, or manually PR only. Thoughts?

**Comment by remiks** [comment]
you are absolutely right! We can build it just as a "generation tool", and all should be submitted through PR for compliance. Everything should be validated before putting it out!
